sos/plugins/docker.py

```diff
@@ -29,20 +29,47 @@ class Docker(Plugin):
         ("all", "enable capture for all containers, even containers "
             "that have terminated", 'fast', False),
         ("logs", "capture logs for running containers",
-            'fast', False)
+            'fast', False),
+        ("size", "capture image sizes for ps", 'slow', False)
     ]
 
     def setup(self):
         self.add_copy_spec([
             "/var/lib/docker/repositories-*"
         ])
 
-        for subcmd in ['info', 'ps', 'ps -a', 'images', 'version']:
+        for subcmd in [
+            'info',
+            'images',
+            'network ls',
+            'ps',
+            'ps -a',
+            'stats --no-stream',
+            'version'
+        ]:
```

Comment:
Maybe make the subcommands list a variable, and then reference it in the subsequent `for` statement?

I know you're following our house style for multi-line lists, but I'm not fond of these in-line with a `for` - the body of the loop ends up looking like it's floating in air...